dts/bindings/adc/nxp,lpc-lpadc.yaml

```diff
@@ -75,6 +75,12 @@ properties:
     type: phandles
     description: References to required regulators which must be enabled for LPADC to function
 
+  ref-supply-val:
```

Comment:
instead of adding a new property, maybe we can make the nxp,reference-supply a phandle array?